src/Components/Host/Host.css

```diff
@@ -27,3 +21,16 @@
     width: 175px;
     padding: 0 15px;
 }
+
+@media only screen and (max-width: 900px) {
+    .hostContainer {
+        padding: 50px;
+    }
+}
+
+@media only screen and (max-width: 600px) {
+    .hostImg {
+        height: 175px;
+        width: 175px;
```

Comment:
the width is already 175px above in `.hostImg`, so this rule is not changing anything